src/App.vue

```diff
@@ -1,31 +1,87 @@
 <template>
-  <div id="app">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet">
-    <router-view/>
+  <div id="app" fluid>
```

Comment:
the "div" tag does not have a "fluid" attribute. Maybe you wanted to use a b-container component?